stories/banner.stories.js

```diff
@@ -234,12 +161,6 @@ export const warning = () => (
   &--Warning {
     @include mercury.Manifold__Banner--Warning;
   }
-
-  ${comment.element}
-
-  &__Icon {
-    @include mercury.Manifold__Banner__Icon;
-  }
```

Comment:
Not a part of this PR, but I noticed there were some lingering things in the Banner HTML no longer needed. I also condensed the SVG code to one line for readability.